README.md

```diff
@@ -118,6 +125,26 @@ end
 
 - `value` - The value to set for the given option. Required for the set action. Booleans are mapped to `on` or `off`.
 
+### gluster\_mountbroker\_user
+
+Use this resource to allow or disallow the given user access to the given volume:
+
+```ruby
+gluster_mountbroker_user 'user/volume_name' do
+  action :add
+end
+```
+
+```ruby
+gluster_mountbroker_user 'user/volume_name' do
+  action :remove
+end
+```
+
+#### Parameters
+
+None.
```

Comment:
If you have another cookbook call this resource with the same name, then you will get a `CHEF-3694` deprecation warning so you need to have this here as an option to avoid that.